### ISSUE TITLE
TF-SLIM : Tensor name "InceptionV3/*/*/*/*/ not found in checkpoint files

### ISSUE THREAD

**Issue by apiszcz** [tensorflow/tensorflow#7775]
Similar problems are in stackoverlow, however they are using flowers example with flowers_train. Using
https://github.com/tensorflow/models/tree/master/slim Fine-tuning a model from an existing checkpoint. The  moving_variance and moving_mean have both had the NotFoundError.

The following train_iamge_classifier

train_image_classifier_nx.py \
--checkpoint_path=data/model/inception-v3/model.ckpt-157585 \
--checkpoint_exclude_scopes=InceptionV3/Logits,InceptionV3/AuxLogits/Logits \
--trainable_scopes=InceptionV3/Logits,InceptionV3/AuxLogits/Logits \
--model_name=inception_v3 \
--train_dir=data/test/train \
--dataset_dir=data/test/traindata

GPU initializes and immediately produces the traceback as follows:

NotFoundError (see above for traceback): Tensor name
"InceptionV3/Mixed_7a/Branch_1/Conv2d_0b_1x7/BatchNorm/moving_variance" not
found in checkpoint files
data/model/inception-v3/model.ckpt-157585
         [[Node: save/RestoreV2_304 = RestoreV2[dtypes=[DT_FLOAT],
         _device="/job:localhost/replica:0/task:0/cpu:0"](_recv_save/Const_0,
         save/RestoreV2_304/tensor_names,
         save/RestoreV2_304/shape_and_slices)]]
                  [[Node: save/RestoreV2_70/_653 =
                  _Recv[client_terminated=false,
                  recv_device="/job:localhost/replica:0/task:0/gpu:0",
                  send_device="/job:localhost/replica:0/task:0/cpu:0",
                  send_device_incarnation=1,
                  tensor_name="edge_1621_save/RestoreV2_70",
                  tensor_type=DT_FLOAT,
                  _device="/job:localhost/replica:0/task:0/gpu:0"]()]]

**Comment by poxvoculi** [comment]
Does this help? https://github.com/tensorflow/tensorflow/issues/4249

**Comment by apiszcz** [comment]
arg_scope is mentioned by others, however i was following the documentation
using the command line train example which has no arg scope option.

**Comment by apiszcz** [comment]
The problem may be due to the fact that the TF-SLIM effort focused on
Python 2.7 and I'm using 3.5 on windows. I just tried the same TF-SLIM with
TF 1.0 and many more issues, it seems few can keep up with all progress on
TF and now with TF1.0 there have been more changes making TF1.0
compatibility with TF-SLIM more challenging. It seems we need a list
stating with contrib/etc. is compatible what version of TF.

**Comment by tfboyd** [comment]
The checkpoints get harder to deal with as TF moves forward.  One example is we would like to add fused batch_norm to slim to speed up training but that will likely further cause issues with the checkpoint.  

I am marking this item as closed as I do not think this will get resolved until we redo this and create new checkpoints.  I know it does not help you but I am sorry that you had trouble.
